pkg/spdk/init.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+#Launch the VTPM server
+mkdir jail; cd jail || exit;
+/usr/bin/vtpm_server
```

Comment:
What's a VTPM server in this context? Does it have anything to do with Trusted Platform Module?

Reply:
It's some residue, will be removed